flowforge/input/System.py

```diff
@@ -112,7 +112,11 @@ def __init__(self, components: Dict[str, Component], sysdict: Dict, unitdict: Di
         self._wallfunctions = []
         self._MMBC = None
         self._EBC = None
+        self._VBC = None
         self._fluid = None
+        self._gas = None
+        self._gasbodyforces = []
+        self._gaswallfunctions = []
```

Comment:
I don't think you need separate body forces / wall functions for gas.  At least not here.  SyTH will split them out when processing `_bodyforces` and `gaswallfunctions`.

Reply:
Agreed

Reply:
removed both